src/docfx/build/dependency/DependencyMap.cs

```diff
@@ -20,14 +21,17 @@ public object ToDependencyMapModel()
         {
             // TODO: Make dependency map a data model once we remove legacy.
             var dependencies = this
-                .OrderBy(d => d.Key.FilePath.Path)
+                .OrderBy(d => GetSourcePath(d.Key.FilePath))
```

Comment:
Can this be computed upfront? calling a method inside `OrderBy` slows things down exponentially

Reply:
could this be a property of `Document`, like `SiteUrl`?